components/02-elements/breadcrumbs/breadcrumbs.hbs

```diff
@@ -1,21 +1,26 @@
-<div class="breadcrumbs {{ class }}" {{{ attributes }}}>
+<ol
+    class="breadcrumbs {{ class }}" {{{ attributes }}}
+    aria-label="{{ ariaLabel }}"
+    role="navigation"
+>
     {{#each categories}}
         {{#if @last}}
-            <span class="breadcrumbs__item"
-                  aria-label="{{ ariaLabel }}"
-                  tabindex="0"
+            <li
+                class="breadcrumbs__item"
+                tabindex="0"
```

Comment:
for last element we should use `<a href="#" aria-current="page">` with a text of last breadcrumb to announce the link of current page :)  and remove tabindex from li